README.md

```
@@ -167,6 +172,18 @@ class MyController
+
+Per-form — declare CSRF on the form itself:
+
+```php
+use Ray\WebFormModule\AbstractForm;
+use Ray\WebFormModule\SetAntiCsrfTrait;
+
+class MyForm extends AbstractForm
+{
+    use SetAntiCsrfTrait;
+}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)